Tests/IntegrationTestProject/ProjectUnderTest/ProjectUnderTest/CSProj1.csproj

```diff
@@ -27,7 +28,7 @@
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <WarningLevel>0</WarningLevel>
```

Comment:
Our current tests perform builds, but do not run Roslyn analyzer code analysis. The E2E Appveyor test does and is the simplest way to assure coverage.